scripts-dev/config-lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
it might be simpler just to grep for the incorrect formats, rather than try to correct them?

Reply:
I considered this at first, but I wanted to be able to add this to `scripts-dev/lint.sh` which could get run in a pre-commit hook for instance.

Reply:
I'm not quite sure that follows... `lint.sh` doesn't correct for incorrect imports or pep8 failures, it just complains if they are wrong.

There might be value in having a script to correct the problems, but given how often it's likely to be a problem versus the added complexity of this script, I don't think it's worthwhile?